recipe/conda_build_config.yaml

```
@@ -0,0 +1,2 @@
+channel_targets:
+  - conda-forge cdat_dev
```

```suggestion
  - conda-forge cdat_info_dev
```
Should't just be `cdat_dev`.  Whould be the package name with `_dev`.

---

@xylar @conda-forge/core  I intentionally used "cdat_dev" because we have several packages  that I am going to rebuild cdat_info, cdtime, cdms2, genutil, cdutil  and I plan to have them uploaded to conda-forge/label/cdat_dev channel (this is for cdat 8.2.1.rc1 metapackage release). @conda-forge/core  Is it an okay practice that I uploaded different packages to cdat/label/cdat_dev channel?